plugins/cache/memcached.py

```diff
@@ -37,13 +38,13 @@
             section: defaults
       _timeout:
         default: 86400
+        type: float
```

Comment:
Why did this change to `float` here? And does it need to also change in ansible-core? - https://docs.ansible.com/ansible/latest/reference_appendices/config.html#cache-plugin-timeout

Reply:
The base class interprets this option as a float: https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/cache/__init__.py#L88

Since other users of that constant might assume it's an int it is probably better not to change the type of `CACHE_PLUGIN_TIMEOUT`, but changing it here should be ok.

@s-hertel you touched that code last, WDYT about using `type: float` here (and the type in general)?

Reply:
I changed it back to `type: integer` for existing `type` values in 46deac0adb3c26a1e12fae3a369ff48abd065e24, only keeping `type: float` when the type wasn't specified before.

Reply:
I'm going to merge this PR in its current form, I can update it back to `type: float` in a follow-up if this discussion resolves positively.